straxen/plugins/raw_records_coin_nv/nveto_recorder.py

```diff
@@ -128,6 +131,30 @@ def compute(self, raw_records_nv, start, end, chunk_i):
                 "lone_raw_record_statistics_nv": lrs,
             }
 
+        # Keep all raw data for the very first n seconds of a run.
+        # This case is tricky since it can also be just a
+        # subset of a chunk and in this case we need to make sure to also keep
+        # all fragments of a pulse beyond the first n seconds boundary.
+        # For performance check very first fragment if in applicable time range:
+        _is_first_record_at_run_start = raw_records_nv[0]["time"] < (
```

Comment:
`_is_first_record_at_run_start` -> `_need_save_for_monitoring` Maybe this is a better name?

Reply:
Done